src/main/java/Calculator.java

```diff
@@ -0,0 +1,21 @@
+public class Calculator {
+
+    public int add(int num1, int num2) {
+        return num1 + num2;
+    }
+
+    public int subtract(int num1, int num2) {
+        return num1 - num2;
+    }
+
+    public int multiply(int num1, int num2) {
+        return num1 * num2;
+    }
+
+    public int divide(int dividend, int divisor) {
+        if(divisor == 0){
+            throw new ArithmeticException("0으로 나눌 수 없습니다.");
+        }
+        return dividend / divisor;

```

Comment:
```suggestion
        if (divisor == 0) {
            throw new ArithmeticException("0으로 나눌 수 없습니다.");
        }
        return dividend / divisor;

```
if문과 소괄호 사이에 공백을 두는 것이 코드 컨벤션입니다! 
저도 처음엔 많이 놓쳤는데, 자동 포맷팅 단축키를 활용하면 편하더라고요. 참고하시면 좋을 것 같아요! 😊

Reply:
좋은 팁 공유해 주셔서 감사합니다 !
GDG과제를 하면서도 코드 컨벤션에 대한 이야기를 코어분들이 많이 해주셨는데, 더 많이 알아보고 
코드에 습관적으로 적용할 수 있게 노력하겠습니다 !